Tools/cases_generator/instructions.py

```diff
@@ -25,18 +25,10 @@ class ActiveCacheEffect:
 
 
 FORBIDDEN_NAMES_IN_UOPS = (
-    "resume_with_error",
-    "kwnames",
     "next_instr",
     "oparg1",  # Proxy for super-instructions like LOAD_FAST_LOAD_FAST
     "JUMPBY",
     "DISPATCH",
-    "INSTRUMENTED_JUMP",
-    "throwflag",
-    "exception_unwind",
-    "import_from",
-    "import_name",
-    "_PyObject_CallNoArgs",  # Proxy for BEFORE_WITH
```

Comment:
@gvanrossum, do we know why we were special-casing `BEFORE_WITH` like this?

I ask because it looks like this PR made `BEFORE_ASYNC_WITH` (which also uses `_PyObject_CallNoArgs`) a viable uop.

Reply:
> @gvanrossum, do we know why we were special-casing `BEFORE_WITH` like this?
> 
> I ask because it looks like this PR made `BEFORE_ASYNC_WITH` (which also uses `_PyObject_CallNoArgs`) a viable uop. It _seems_ fine, but maybe I'm missing something.

Urgh, I have no idea. :-( Agreed that both seem fine. Either both should have the `TIER_ONE_ONLY` marker, or neither should. Let me dig through some history:

It's been like this since the cases_generator code first landed (#105924).  And even then, commenting out that line makes `BEFORE_*WITH` appear in the output and it compiles just fine. In the PR I found this commit though:
https://github.com/python/cpython/pull/105924/commits/e3471dabacd0a6b488b7da346f13f60bb5a1edf8

The commit message gives a clue:

> Fix test_memoryview by excluding BEFORE_WITH
> 
> Unclear what's wrong with that opcode."

Commenting out the line, regenerating, and then running `./python.exe -Xuops -m test test_memoryview` gives this error:
```
test test_memoryview failed -- Traceback (most recent call last):
  File "/Users/guido/cpython/Lib/test/test_memoryview.py", line 288, in test_contextmanager
    self._check_released(m, tp)
  File "/Users/guido/cpython/Lib/test/test_memoryview.py", line 273, in _check_released
    with m:
         ^
ValueError: operation forbidden on released memoryview object
```
Doing the same on main, however, causes no crashes. Quite possibly whatever was defective in the original code generator and Tier 2 interpreter has since been fixed, and the TIER_ONE_ONLY marker can safely be removed from BEFORE_WITH. I don't feel like doing a bisection to figure out the exact commit that made the test_memoryview failure go away.